pages/common/chromium.md

```diff
@@ -1,15 +1,11 @@
 # chromium
 
-> Open-source web browser from Google.
+> Open-source web browser principally developed and maintained by Google.
 > More information: <https://chromium.org>.
 
-- Open a file:
+- Open a specific URL or file:
 
-`chromium {{path/to/file.html}}`
-
-- Open a URL:
-
-`chromium {{example.com}}`
+`chromium {{https://example.com|path/to/file.html}}`
 
 - Open in incognito mode:
```

Comment:
```suggestion
- Open in an incognito mode:
```

Reply:
not sure about this :thinking: I think the original description has a better grammar.

Reply:
I think the original here makes more sense. 👍🏻